### PR TITLE
Ear timestamp

### PR DESCRIPTION
**Pull request by ear-dev**
@Shailesh351 this PR just adds a new package to create a timestamp at the beginning of every log point.

### CODE REVIEW

**Comment by Shailesh351** [comment]
@ear-dev build is failing because of the lint issue. Also, there are some other changes in package-lock.json file, which are old changes. Can you rebase your changes on top of develop_pwa

**Comment by ear-dev** [comment]
Thanks @Shailesh351 

Can you take a look at package-lock.json please?  I'm not sure if it still has old stuff in it. Thanks.

File: package-lock.json [hunk 1]
```diff
@@ -5907,7 +5907,7 @@
 			}
 		},
 		"@rocket.chat/apps-engine": {
-			"version": "git+https://github.com/WideChat/Rocket.Chat.Apps-engine.git#124d506b843adf24dedc1a8fbdee5e38583ab710",
+			"version": "git+https://github.com/WideChat/Rocket.Chat.Apps-engine.git#0ebf68c094944ef719834e81fab7b63700d86305",
```
Comment (Shailesh351) [inline package-lock.json:5910]: @ear-dev Still I see these changes. It's older apps-engine commit id.

**Comment by Shailesh351** [comment]
@ear-dev I have created this PR https://github.com/WideChat/Rocket.Chat/pull/589 for same changes

**Pull request closed without merging**